integration-tests/controller-scripts/package.json

```diff
@@ -3,8 +3,7 @@
   "version": "1.0.0",
   "description": "",
   "scripts": {
-    "start": "node index.js",
-    "html:report": "for dir in $(ls reports);do junit2html ./reports/$dir/report.xml --out ./reports/$dir/report.html;done"
+    "start": "node index.js"
```

Comment:
Why don't we leave this as `"run": "node resultsProcessor.js"` and then just execute `npm run` when executing this project instead?

Reply:
Missed that, good point!